woocommerce/woocommerce-android
# Delete RefundByAmount fragment

First PR for #3489 

This PR removes all code related to RefundByAmount feature which is not used anymore due to a bug on the server - more info p91TBi-3hR#comment-1762-p2. This PR doesn't remove IssueRefundFragment since I wanted to keep these two changes separate to make the review a bit easier.

I'm new to this part of the app and I'm not sure if it can be in a different state than the following: (in other words, I haven't tested anything else but the following flow)
| <img width="274" alt="Screenshot 2021-03-17 at 8 46 05" src="https://user-images.githubusercontent.com/2261188/111486800-68523d00-86fd-11eb-8b2c-5ba10950b074.png"> | <img width="272" alt="Screenshot 2021-03-17 at 8 46 16" src="https://user-images.githubusercontent.com/2261188/111486810-6b4d2d80-86fd-11eb-9a62-98a23c1dc7fd.png"> | <img width="273" alt="Screenshot 2021-03-17 at 8 46 22" src="https://user-images.githubusercontent.com/2261188/111486834-71dba500-86fd-11eb-8549-fdd2e9de3975.png"> |
|---|---|---|


To test:
1. Tap on Orders
2. Select an order
3. Tap on Issue Refund
4. Make sure the Refund flow works - there should be no user facing changes.


Update release notes:
